controllers/users.js

```diff
@@ -37,12 +37,13 @@ class Users {
         status: 201,
         message: 'user created',
         user: {
+          id: user.id,
           email: user.email,
           username: user.username
         }
       });
     } catch (e) {
-      return e;
+      return res.send({ error: e });
```

Comment:
You can avoid generic error return if possible.

Reply:
solved